afs/templates/views/components/reports/project.htm

```diff
@@ -73,6 +73,51 @@ <h1 class="afs-report-toolbar-title"><span class="afs-report-name">{% trans "Pro
                     }
                 }"></div>
             </div>
+            <!-- Component Project Tab -->
+            <div class="afs-report-page" data-bind="if: activeSection() === 'components'">
+                <div class="afs-report-section">
+                    <h2 class="afs-report-section-title"><i data-bind="click: function() {toggleVisibility(visible.childProjects)}, css: {'fa-angle-double-right': visible.childProjects(), 'fa-angle-double-up': !visible.childProjects()}" class="fa toggle"></i> {% trans "Child Projects" %}</h2>
+                    <!-- Collapsible content -->
+                    <div data-bind="visible: visible.childProjects" class="afs-report-collapsible-container pad-lft">
+
+                        <!-- ko ifnot: collectionOfChildProjects().length -->
+                        <div class="afs-nodata-note">No child projects for this project</div>
+                        <!-- /ko -->
+                        
+                        <!-- ko if: collectionOfChildProjects().length -->
+                        <div class="afs-report-subsection" >
+                            <div>
+                                <div class="afs-table pad-btm">
+                                    <table class="table table-striped" cellspacing="0" width="100%">
+                                        <thead>
+                                            <tr class="afs-table-header">
+                                                <th>{% trans "Project" %}</th>
+                                                <th class="min-tabletl">{% trans "Description" %}</th>
+                                                <th class="none">{% trans "Visit Resource" %}</th>
```

Comment:
This extra column needs to be removed - your th count needs to match your td count.  Check your chrome console - there's an error in there being caused by this.